core/src/main/java/com/linecorp/armeria/client/HttpRequestSubscriber.java

```diff
@@ -297,6 +296,16 @@ private void write(HttpObject o, boolean endOfStream, boolean flush) {
     }
 
     private void write0(HttpObject o, boolean endOfStream, boolean flush) {
+        // if the first bytes transferred, the stream must be open
```

Comment:
Could you explain why?

Reply:
The reason why I'm asking this is because the comment itself simply explains what the code does. It does not add much to the code without the comment in my opinion.

If we think there's something more to explain about the code, it has to be more detailed than this.

Reply:
I agree with your optinion. I will add the detail explanation about 
* Why I use `loggedRequestFirstBytesTransferred` to check stream condition
* When this method could be called even if the stream is closed.